### PR TITLE
Use GitHub Actions Greetings and Stale

### PR DESCRIPTION
**Pull request by nemchik**
https://github.com/actions/starter-workflows/blob/e11672e5a9026a98ce6235af089d3112ec93dbd5/automation/greetings.yml
https://github.com/actions/first-interaction

This will comment on the first issue and first PR anyone opens in any of our repos (so possibly 2 comments in every repo that a user interacts with). I set the message to encourage users to use the respective template for the repo.

https://github.com/actions/starter-workflows/blob/e11672e5a9026a98ce6235af089d3112ec93dbd5/automation/stale.yml
https://github.com/actions/stale

This will add a label to issues and pull requests that are considered stale after a period of inactivity. After another period the issue/pr will be closed. I'll bring this up in chat to discuss additional options (inactivity period, exempt labels, etc).

Per discussions in chat:

- `no-issue-activity` and `no-pr-activity` will be added (by the bot) to issues and PRs that become stale
- Issues and PRs will become stale after **30** days of no activity
- Stale issues and PRs will be closed (by the bot) **365** after becoming stale
- Issues and PRs with the label `awaiting-approval` or `work-in-progress` (applied/created manually if needed) will not be considered stale
- A semi-long term goal will be to reduce the **365** to a lower number (the default is **5**) over time as we adjust to the bot and the labels

### CODE REVIEW

**Comment by thelamer** [comment]
@nemchik when this gets merged please also merge https://github.com/linuxserver/docker-jenkins-builder/pull/49 so all the logic waterfalls

File: roles/generate-jenkins/templates/Jenkinsfile.j2 [hunk 1]
```diff
@@ -391,7 +391,7 @@ pipeline {
 {% if github_project_name != "docker-jenkins-builder" %}
               docker pull linuxserver/jenkins-builder:latest
 {% endif %}
-              docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %}
```
Comment (thelamer) [inline roles/generate-jenkins/templates/Jenkinsfile.j2:394]: Check for these spaces they are needed for templating

Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:394]: Would this work?
```suggestion
docker run --rm -e CONTAINER_NAME=${CONTAINER_NAME} -e GITHUB_BRANCH={{ ls_branch }} -v ${TEMPDIR}:/ansible/jenkins {% if github_project_name != "docker-jenkins-builder" %}linuxserver/jenkins-builder:latest{% else %}jenkinslocal:${COMMIT_SHA}-${BUILD_NUMBER}{% endif %} {{ ' ' }}
```
Ref https://jinja.palletsprojects.com/en/2.11.x/templates/#escaping

Comment (thelamer) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:394]: not sure you can build and template locally

Comment (thelamer) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:394]: it is about the line break though not the space

Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:394]: I'll test add report.

Comment (nemchik) [reply roles/generate-jenkins/templates/Jenkinsfile.j2:394]: Tested and confirmed.

**Pull request merged**